### ISSUE TITLE
Allow the user to choose what buttons are shown in the Verse Action Dialog

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1812]
I would like to have the 'Speak' button in that list. It used to be there once upon a time :)
I never use the 'My Notes' button.
I would love to be able to get to Search more directly as well.
I would like the option of adding more buttons to that list.

Perhaps this could become customisable by allowing the user to chose what buttons they want.

**Comment by tuomas2** [comment]
> I would like to have the 'Speak' button in that list. It used to be there once upon a time :)

It was removed in 8eb8803477d65eb9a930ebd973d0358df2b21c42 due to a bug that was not fixed. #1431

**Comment by bvahedy** [comment]
Yes, I also loved having the search and speak in the quick action menu.
